dbsa/presto.py

```diff
@@ -166,3 +166,33 @@ def get_create_current_partition_view(self, suffix='_latest', condition='', igno
             select=self.get_select_current_partition(condition=condition, ignored_partitions=ignored_partitions, params=params, transforms=transforms),
             suffix=suffix,
         )
+
+    def get_incremental_update_select(self, update_select, row_identifier=None, filter_fn=None, condition='', ignored_partitions=None, params=None, transforms=None):
+        return Template("""
+            WITH incremental_update AS (
+                {{ update_select }}
+            ),
+            recent_data AS (
+                {{ select }}
+            )
+            SELECT *
+            FROM incremental_update
+            UNION ALL
+            SELECT * 
+            FROM recent_data
+            {%- if row_identifier %}
+            WHERE {{ row_identifier }} NOT IN (
```

Comment:
We should do `NOT EXISTS` to avoid potential problems via NULLs. 

Also using NOT EXISTS would open up the possibility to support multi-column PKs